patches/sqlite/0001-row-counts-plain.patch

```diff
@@ -1,26 +1,199 @@
-diff -u5 -r sqlite-src-3440000.pristine/src/shell.c.in sqlite-src-3440000/src/shell.c.in
---- sqlite-src-3440000.pristine/src/shell.c.in	2023-11-01 04:31:37.000000000 -0700
-+++ sqlite-src-3440000/src/shell.c.in	2023-11-03 15:06:27.994222697 -0700
-@@ -3381,10 +3381,15 @@
-     raw_printf(pArg->out, "Reprepare operations:                %d\n", iCur);
-     iCur = sqlite3_stmt_status(pArg->pStmt, SQLITE_STMTSTATUS_RUN, bReset);
-     raw_printf(pArg->out, "Number of times run:                 %d\n", iCur);
+diff -u5 -r sqlite-src-pristine/src/shell.c.in sqlite-src-modified/src/shell.c.in
```

Comment:
This patch has ballooned in size and it shouldn't have.  As far as I can tell, the main changes should be changing out 2 `raw_printf` calls for 2 `sqlite3_printf`.

It looks like your editor removed trailing whitespace.  We shouldn't be using patches to remove trailing whitespace from SQLite.

It also looks like the ordering of src/vdbeapi.c and src/vdbe.c got reversed.  Can we put them back in the order they used to be?  (Or re-order them in a separate commit.)

Reply:
Fair point about the whitespace. I assumed that this was a difference in versions between patch. `src/vdbeapi.c` and `src/vdbe.c` were not manually reversed. I'd rather keep the output of patch as a single commit rather than risk fat fingering something.